docs/source/commands/license.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: The Rover license command
+description: Manage Enterprise licenses 
+---
+
+Rover commands that manage Enterprise licenses for Apollo begin with `rover license`.
+
+## Fetch offline Enterprise license 
+
+### `license fetch`
+
+Run `rover license fetch` to fetch a license from GraphOS for a graph with a GraphOS Enterprise plan.
+
+Specify the graph to fetch from with a `--graph-id` argument: 
+
+```bash
+rover license fetch --graph-id <apollo-graph-id>
+```
+
+<Note>
+
+* An offline Enterprise license is valid for the lesser of the duration of your contract with Apollo, or one year, with an added grace period of 28 days. 
+
+* You are responsible for keeping your offline license files up to date within your infrastructure.
```

Comment:
I couldn't find anything about "keeping your offline license files up to date within your infrastructure" in https://github.com/apollographql/router/pull/4327/files ... Does this phrase just mean running the `rover license fetch` command at least once a year? If so, maybe something like this for clarity:

```suggestion
* You are responsible for keeping your offline license files up to date within your infrastructure by running `rover license fetch` at least once a year.
```

Also, is offline license expiration something that's worth mentioning in the router docs as well?